README.md

```diff
@@ -1 +1,2 @@
-# firstrepo
\ No newline at end of file
+# firstrepo
+# change 1 only on branch1
```

Comment:
say instead "change 1 by jltabc (and not by jltabcGmail)only on branch1"

Reply:
taken action